File: app/src/androidTest/java/ua/sytor/wifipass/ApplicationTest.java
package ua.sytor.wifipass;

import org.junit.Assert;
import org.junit.Test;
import ua.sytor.wifipass.interactor.command_executer.CommandExecutorContract;
import ua.sytor.wifipass.interactor.command_executer.CommandExecutor;

public class ApplicationTest {

    CommandExecutorContract commandExecutor = new CommandExecutor();

    @Test
    public void testCmdCommand() {

        String output = null;
        try {
            output = commandExecutor.execCommand("su -c cat /data/misc/wifi/wpa_supplicant.conf");
        } catch (Exception e) {
            e.printStackTrace();
        }
        System.out.println("output " + output);
        Assert.assertEquals(1, 1);

    }

    @Test
    public void testCommand() {

    }

    @Test
    public void testSu() {

    }

    @Test
    public void getContent() {

    }

}

File: app/src/androidTest/java/ua/sytor/wifipass/ApplicationTest.kt
package ua.sytor.wifipass

import org.junit.Assert
import org.junit.Test
import ua.sytor.wifipass.core.command_executer.CommandExecutor
import ua.sytor.wifipass.core.command_executer.CommandExecutorContract

class ApplicationTest {

	var commandExecutor: CommandExecutorContract.CommandExecutor = CommandExecutor()

	@Test
	fun testCmdCommand() {
		var output: String? = null
		try {
			output = commandExecutor.execCommand("su -c cat /data/misc/wifi/wpa_supplicant.conf")
		} catch (e: Exception) {
			e.printStackTrace()
		}
		println("output $output")
		Assert.assertEquals(1, 1)
	}

	@Test
	fun testCommand() {

	}

	@Test
	fun testSu() {
	}

}